data/frontier jobs.txt

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) 2017 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Frontier delivery [0]"
+	name "Frontier delivery to <planet>"
+	job
+	repeat
+	description `Deliver <cargo> to the frontier world of <destination>. Frontier worlds have a high rate of piracy, so a pirate presence is expected. Payment is <payment>.`
```

Comment:
[for all of these jobs]

I don't like "pirate" and "piracy" in close proximity. Maybe change this sentence to "Frontier worlds have a high rate of piracy, so be prepared for attack." (or something like that)